Doc/c-api/unicode.rst

```diff
@@ -854,7 +854,8 @@ wchar_t Support
    Copy the Unicode object contents into the :c:type:`wchar_t` buffer *wstr*.  At most
    *size* :c:type:`wchar_t` characters are copied (excluding a possibly trailing
    null termination character).  Return the number of :c:type:`wchar_t` characters
-   copied or ``-1`` in case of an error.  Note that the resulting :c:expr:`wchar_t*`
+   copied or ``-1`` in case of an error. When :c:type:`wchar_t` buffer *wstr* is NULL, return 
+   the length of the Unicode object plus one.Note that the resulting :c:expr:`wchar_t*`
```

Comment:
Careful: the `len()` of an unicode object might not match the length of a `wchar_t` buffer with its data.
IMO we need something like this; maybe it could be made more terse:
```suggestion
   copied or ``-1`` in case of an error.

   When *wstr* is ``NULL``, instead return the *size* that would be required
   to store all of *unicode* including a terminating null.

   Note that the resulting :c:expr:`wchar_t*`
```